lib/features/modeling/ChoreoModeling.js

```diff
@@ -108,4 +110,16 @@ ChoreoModeling.prototype.changeParticipant = function(bandShape, participant) {
     bandShape: bandShape,
     newParticipant: participant
   });
+};
+
+ChoreoModeling.prototype.setInitiatingBand = function(element) {
```

Comment:
Might be better to rename `element` to `bandShape`, it took me a few seconds to get that it's a band.